elements/lisk-p2p/src/peer/outbound.ts

```diff
@@ -47,6 +47,11 @@ export const EVENT_CLOSE_OUTBOUND = 'closeOutbound';
 export const EVENT_OUTBOUND_SOCKET_ERROR = 'outboundSocketError';
 export const RESPONSE_PONG = 'pong';
 
+const socketErrorStatusCodes = {
+	...(socketClusterClient.SCClientSocket as any).errorStatuses,
```

Comment:
same here

Reply:
As above, it will always exist. I had to do this casting because the DefinitelyTyped definition is incomplete.